index.js

```diff
@@ -183,6 +184,41 @@ class GithubScm extends Scm {
         };
     }
 
+    /**
+     * Promise to wait a certain number of seconds
+     *
+     * Might make this centralized for other tests to leverage
+     *
+     * @method promiseToWait
+     * @param  {Number}      timeToWait  Number of seconds to wait before continuing the chain
+     * @return {Promise}
+     */
+    promiseToWait(timeToWait) {
+        return new Promise((resolve) => {
+            setTimeout(() => resolve(), timeToWait * 1000);
+        });
+    }
+
+    /**
+     * Get PR mergeability
+     * @async  getPrMergeable
+     * @param  {Object}   config
+     * @param  {String}   config.scmUri     The scmUri to get PR info of
+     * @param  {String}   config.token      The token used to authenticate to the SCM
+     * @param  {Integer}  config.prNum      The PR number used to fetch the PR
+     * @return {Promise}
+     */
+    async getPrMergeable({ scmUri, token, prNum }) {
+        const { mergeable } = await this.getPrInfo({ scmUri, token, prNum });
```

Comment:
Is there any case that `mergeable` is always null?
If so, it seems that this loop is endless.

Reply:
It seems that `mergeable` is null until finishing to compute the mergeability.

https://developer.github.com/v3/pulls/
`If the value is null, then GitHub has started a background job to compute the mergeability. `

Reply:
I see. Finally, the `mergeable` will be `true` or `false` definitely.

Reply:
Do we need try-catch statement?
If any error cause, that error may not be caught.
https://github.com/screwdriver-cd/scm-github/blob/f6de4a4c88a696f496d62d3c5efcc7daa3c1e7b4/index.js#L1313

Reply:
Thanks. I fixed.